app/controllers/application_controller.rb

```diff
@@ -40,7 +40,7 @@ def ldap_groups
   def groups_from_request_env
     session['groups'] = begin
       raw_header = request.env[Settings.authorization_group_header]
-      raw_header = ENV['ROLES'] if Rails.env.development? # rubocop:disable Rails/EnvironmentVariableAccess
+      raw_header = ENV.fetch('ROLES', nil) if Rails.env.development? # rubocop:disable Rails/EnvironmentVariableAccess
```

Comment:
AFAICT, this *should* operate the same as it did before. Can you say more about this change?

Reply:
this is due to Style/FetchVar cop;  it's to avoid inadvertent nil coming from ENV vars, I think.